solver/exec.go

```diff
@@ -84,9 +85,17 @@ func (e *execOp) Run(ctx context.Context, inputs []Reference) ([]Reference, erro
 			outputs = append(outputs, active)
 			mountable = active
 		}
-		mounts[m.Dest] = mountable
+		if m.Dest == "/" {
```

Comment:
This isn't really platform-specific. It is more like a constant how a mount that should be used as rootfs is described in llb. I expect it to be `/` in windows as well (or maybe an actual constant or property).